src/scripts/main.js

```diff
@@ -1 +1,10 @@
 'use strict';
+
+const galleryList = document.querySelector('ul');
+const largeImg = document.querySelector('#largeImg');
+
+galleryList.addEventListener('click', (even) => {
+  even.preventDefault();
+  largeImg.src = even.target.src;
```

Comment:
take the image from `href` attribute of `a` tag
it will prevent changes in size after clicking (in this case)